lib/generate-report.js

```diff
@@ -49,13 +49,14 @@ function generateReport(options) {
 
 	const customMetadata = options.customMetadata || false;
 	const customData = options.customData || null;
+	const plainDescription = options.plainDescription || false;
```

Comment:
Converts Object property to boolean with a shorthand. If it was falsey (e.g. 0, null, undefined, etc.), it will be false, otherwise, true.

```suggestion
	const plainDescription = !!options.plainDescription;
```

Reply:
Can you also do the above for

```js
const disableLog = options.disableLog || false;
const openReportInBrowser = options.openReportInBrowser || false;
const saveCollectedJSON = options.saveCollectedJSON || false;
const displayDuration = !!options.displayDuration;
const displayReportTime = options.displayReportTime || false;
const durationInMS = options.durationInMS || false;
const hideMetadata = options.hideMetadata || false;
const pageFooter = options.pageFooter || false;
const useCDN = options.useCDN || false;
const staticFilePath = options.staticFilePath || false;
```

Reply:
Ahh...the `truthy`/`falsy` always gets me. :facepalm: